openlr/candidate_paths_getter.hpp

```diff
@@ -87,9 +87,9 @@ class CandidatePathsGetter
 
     LinkPtr m_path = nullptr;
     uint32_t m_bearingDiff = std::numeric_limits<uint32_t>::max();  // Domain is roughly [0, 30]
-    uint32_t m_pathDistanceDiff =
+    double m_pathDistanceDiff =
```

Comment:
Are you sure that initialization is right? Why not to replace this to double max or to inf?

Reply:
My bad, I missed this one.

Reply:
Done.